docs/MapColonies/vector/Services/feedback-api/README.md

```diff
@@ -0,0 +1,140 @@
+---
+id: vector-feedback-api-service
+slug: info
+title: Geocoding's Feedback Api ⚙️
+description: Feedback of Geocoding's responses
+tags:
+  - vector
+  - API
+  - osm
+  - Geocoding
+---
+
+# Geocoding's Feedback Api ⚙️
+
+:::info
+Click [here](/docs/MapColonies/vector/services/feedback-api/api) for the OpenAPI
+:::
+
+:::note
+If you're not familier with Geocoding API, [read first here](/docs/MapColonies/vector/Services/geocoding/README.md).
+:::
+
+Geocoding's feedback api is an API that gets from the user of [Geocoding API](/docs/MapColonies/vector/Services/geocoding/README.md) the response and and puts it in redis for BI purposes.
+
+## Usage
+
+:::caution
+**You will need an API token as part of the [service authentication](/docs/MapColonies/authentication). &nbsp;**
+:::
+
+Once a user gets a response from Geocoding, the system that uses Geocoding will send the request ID and the chosen response ID back to us using the API.
```

Comment:
Please move this to the section before `usage` and consider removing this section.

Reply:
You mean just remove the usage title?